cmd/ocm-backplane/console/console.go

```diff
@@ -214,6 +217,168 @@ func checkAndFindContainerURL(containerName string, containerEngine string) (err
 	return nil
 }
 
+func getBackplaneConfigDirctory() (string, error) {
+	bpConfigFilePath, err := config.GetConfigFilePath()
+	if err != nil {
+		return "", err
+	}
+	configDirectory := filepath.Dir(bpConfigFilePath)
+
+	return configDirectory, nil
+}
+
+// setupMonitoringNginxConfig sets up a nginx configuration because
+// the monitoring container runs a nginx to accept requests
+func setupMonitoringNginxConfig(port string) error {
+	// setup the config file path
+	configDirectory, err := getBackplaneConfigDirctory()
+	if err != nil {
+		return err
+	}
+
+	configFilename := filepath.Join(configDirectory, "monitoring-plugin-nginx.conf")
+
+	// Check if file already exists, if it does remove it
+	if _, err = os.Stat(configFilename); !os.IsNotExist(err) {
+		err = os.Remove(configFilename)
+		if err != nil {
+			return err
+		}
+	}
+
+	configTemplate := `
+error_log /dev/stdout info;
+events {}
+http {
+  include            /etc/nginx/mime.types;
+  default_type       application/octet-stream;
+  keepalive_timeout  65;
+  server {
+    listen              %s;
+    root                /usr/share/nginx/html;
+  }
+}
+	`
+
+	config := fmt.Sprintf(configTemplate, port)
+
+	err = os.WriteFile(configFilename, []byte(config), 0600)
+	if err != nil {
+		return err
+	}
+
+	// change permission as a work around to gosec
+	err = os.Chmod(configFilename, 0640)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// runMonitoringContainer runs the monitoring plugin in the same network as the console container
+func runMonitoringContainer(name string, configFilename string, containerEngine string, config *rest.Config) error {
+	logger.Debugln("Querying the cluster for monitoring-plugin image")
+	image, err := getMonitoringPluginImageFromCluster(config)
+	if err != nil {
+		return err
+	}
+	logger.Infof("Using monitoring-plugin image %s\n", image)
+
+	// get the monitoring plugin nginx config
+	configDirectory, err := getBackplaneConfigDirctory()
+	if err != nil {
+		return err
+	}
+	configFilePath := filepath.Join(configDirectory, "monitoring-plugin-nginx.conf")
+
+	// Check if file already exists, if it does not, stop
+	if _, err = os.Stat(configFilePath); !os.IsNotExist(err) {
+		if err != nil {
+			return fmt.Errorf("failed to find the monitoring plugin nginx configuration: %s", err)
+		}
+	}
```

Comment:
What if define the file name as constant in[ info.go](https://github.com/openshift/backplane-cli/blob/main/pkg/info/info.go) , 
is it good to move this logic as function into `pkg/cli/config`  package ?